addons/mass_mailing/views/email_template.xml

```diff
@@ -84,6 +84,13 @@
     'form_view_ref': 'mass_mailing.email_template_form_minimal',
     'default_use_default_to': True,
 }</field>
+            <field name="help" type="html">
+                <p class="oe_view_nocontent_create">
+                    Click here to create a new Template.
+                </p> <p>
+                    Mailing Templates allow you to easily design template which used when send mail.
```

Comment:
Mail templates are meant to design e-mails that can be used for several mass mailing campaigns.